### PR TITLE
adding antorbital notch

### PR DESCRIPTION
**Pull request by megbalk**
from issue #1886

### CODE REVIEW

**Comment by megbalk** [comment]
What is exit code 2?

**Comment by paolaroncaglia** [comment]
Could anyone assist @megbalk with her question please: 

> What is exit code 2?

(Meghan, you may also wish to try doing the pull request again if the error came out of the blue)

**Comment by megbalk** [comment]
I tried doing re-doing the pull request and got the same error. @cmungall or @nicolevasilevsky could you help? I don't know what "exit code 2" error is.

**Comment by nicolevasilevsky** [comment]
where do you see that error code?

**Comment by megbalk** [comment]
Clicking on details under "all checks have failed".
Screen Shot 2022-01-06 at 11 22 20 AM

**Comment by balhoff** [comment]
You need to look at the next section in the checks output—here is the real error:

> OBO STRUCTURE ERROR Ontology does not conform to OBO structure rules:
17
multiple name tags not allowed. in frame:Frame(UBERON:7500070 id( UBERON:7500070)name( antorbital notch)property_value( http://purl.org/dc/elements/1.1/date 2021-05-28T18:26:38Z xsd:dateTime)namespace( uberon)name( sulcus tali)def( The antorbital notch is a prominence of bone located anteriolaterally of the antorbital fenestra.)def( A depression that runs between the two facets of a talus that interact with a calcaneum.[https://en.wikipedia.org/wiki/Talus_bone ])property_value( term:editor http://orcid.org/0000-0003-2699-3066)property_value( present_in_taxon Equus xsd:string)property_value( present_in_taxon Cetacea xsd:string)synonym( sulcus for flexor hallucis longus tendon EXACT)property_value( http://purl.org/dc/elements/1.1/date 2021-05-28T01:09:52Z xsd:dateTime)relationship( part_of UBERON:0002395)is_a( UBERON:0006846)relationship( part_of UBERON:0003129)is_a( UBERON:0000093))
18
For details see: http://robot.obolibrary.org/errors#obo-structure-error

**Comment by megbalk** [comment]
Thanks @balhoff, I think I've fixed the issue

**Comment by megbalk** [comment]
@paolaroncaglia can you review this?

**Review by paolaroncaglia — COMMENTED** [review]
Hi @megbalk ,
In the definition, is "anteriolaterally" a misspelling of "anterolaterally"? If it's correct but just less used, it's ok to leave. 
Ideally, you'd want a reference in support of a new term, e.g. a PMID or ISBN or Wikipedia link as an annotation property (dbxref) of the definition. But if none is easily available, your ORCID will serve the purpose.
I see your Protege preferences are set to use term:editor, mine use a different property; I remember there was some discussion to adopt a standard, but I'm just back from sick leave and can't remember exactly. Perhaps @nicolevasilevsky may comment please if term:editor should not be used - otherwise I think it's acceptable. Thanks.

**Comment by megbalk** [comment]
@paolaroncaglia thank you for catching those!
@nicolevasilevsky could you help if I should use editor or creator?

**Comment by nicolevasilevsky** [comment]
I'm using this creator property:
http://www.geneontology.org/formats/oboInOwl#created_by

image

I think that is the standard we agreed on.

**Comment by megbalk** [comment]
fixed! @paolaroncaglia let me know if it looks good.

**Comment by paolaroncaglia** [comment]
@megbalk PR looks good to me, but the checks are failing again :-(

**Comment by megbalk** [comment]
it says "Makefile:563: recipe for target 'sparql_test' failed". @balhoff @dosumis could you help me understand this?

**Comment by balhoff** [comment]
The problem is `FAIL Rule ../sparql/illegal-annotation-property-violation.sparql: 1 violation(s)`. But it doesn't print which property. Maybe it's your `date` property?

**Comment by megbalk** [comment]
@paolaroncaglia it looks like it's passed! could you please approve?

**Comment by megbalk** [comment]
@paolaroncaglia it looks like it passed! Could you please approve?

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224519,6 +224519,16 @@ is_a: UBERON:7500094 ! tubercle of calcaneus
...
 property_value: term:editor http://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500101
+name: antorbital notch
+def: "The antorbital notch is a prominence of bone located anterolaterally of the antorbital fenestra." [doi:10.5479/si.00810282.627]
+is_a: UBERON:0006846 ! surface groove
+relationship: part_of UBERON:0003129 ! skull
+property_value: present_in_taxon "Cetacea" xsd:string
+property_value: present_in_taxon "Equus" xsd:string
```
Comment (dosumis) [inline src/ontology/uberon-edit.obo:224529]: These should link to the terms in NCBI_Taxon, rather than being strings.

**Review by dosumis — CHANGES_REQUESTED** [review]
See note on present_in_taxon axioms

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224529]: @dosumis I don't see an option for that in the annotation properties.

Comment (megbalk) [reply src/ontology/uberon-edit.obo:224529]: @dosumis how do I do that?

Comment (dosumis) [reply src/ontology/uberon-edit.obo:224529]: Just realised that Protege doesn't make it easy as you can't use the class expression editor for an AP value.
- you can  navigate to the term by hand like this - which is not very practical
image
- much easier is to search for the term 
image
copy the IRI (use COMMAND U to display) 
image
then paste into the IRI editor on the term you're editing:
image
=>
image

**Review by dosumis — APPROVED** [review]

**Pull request merged**